src/ui/App.hpp

```diff
@@ -35,7 +38,8 @@ class App {
 public:
     std::string executable;
 
-    std::unique_ptr<Dashboard> dashboard;
+    std::unique_ptr<Dashboard>                   dashboard;
+    std::shared_ptr<opencmw::client::RestClient> restClient = std::make_unique<opencmw::client::RestClient>(opencmw::client::VerifyServerCertificates(Digitizer::Settings::instance().checkCertificates));
```

Comment:
Ideally this client would also be used in the RemoteSource blocks, so we have one connection only per remote peer.